File: app/src/main/java/dev/ragnarok/fenrir/api/model/VkApiMarketAlbum.java
package dev.ragnarok.fenrir.api.model;

import androidx.annotation.NonNull;

import com.google.gson.annotations.SerializedName;


public class VKApiMarketAlbum implements VKApiAttachment {

    @SerializedName("id")
    public int id;

    @SerializedName("owner_id")
    public int owner_id;

    @SerializedName("access_key")
    public String access_key;

    @SerializedName("title")
    public String title;

    @SerializedName("photo")
    public VKApiPhoto photo;

    @SerializedName("count")
    public int count;

    @SerializedName("updated_time")
    public int updated_time;

    @NonNull
    @Override
    public String getType() {
        return TYPE_MARKET_ALBUM;
    }

}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/VKApiMarketAlbum.kt
package dev.ragnarok.fenrir.api.model

import com.google.gson.annotations.SerializedName

class VKApiMarketAlbum : VKApiAttachment {
    @SerializedName("id")
    var id = 0

    @SerializedName("owner_id")
    var owner_id = 0

    @SerializedName("access_key")
    var access_key: String? = null

    @SerializedName("title")
    var title: String? = null

    @SerializedName("photo")
    var photo: VKApiPhoto? = null

    @SerializedName("count")
    var count = 0

    @SerializedName("updated_time")
    var updated_time: Long = 0
    override fun getType(): String {
        return VKApiAttachment.TYPE_MARKET_ALBUM
    }
}